test/end-to-end/pages/IDAM/signIn.js

```diff
@@ -11,11 +11,12 @@ module.exports = async function (noScreenerQuestions = false) {
             await I.amOnLoadedPage('/');
         }
 
+        await I.wait(3);
         const locator = {css: 'a[href="/sign-out"]'};
```

Comment:
do we need this explicit wait?

Reply:
good spot, that was just for testing, will fix 👍

Reply:
done